_data/projects/basic/awkward-array.yml

```diff
@@ -1,8 +1,8 @@
 name: awkward-array
 projlogo: /assets/images/projlogo/logo_awkward_array.png
 description: Manipulate arrays of complex data structures as easily as Numpy.
-url: https://github.com/scikit-hep/awkward-array
-readme: https://github.com/scikit-hep/awkward-array/blob/master/README.rst
+url: https://github.com/scikit-hep/awkward-array-1.0
+readme: https://github.com/scikit-hep/awkward-array-1.0/blob/master/README.rst
```

Comment:
```suggestion
readme: https://github.com/scikit-hep/awkward-array-1.0/blob/main/README.rst
```

Reply:
I don't know how to suggest to a suggestion, so I'll have to write it manually here:

```
https://github.com/scikit-hep/awkward-1.0/#readme
```

The repo name is `awkward-1.0`, not `awkward-array-1.0` and links to the `<div id="readme">` element are more useful because the focus starts on the README, but users can scroll up to see the normal front-page stuff.